### ISSUE TITLE
Better control over minimum and maximize buttons

### ISSUE THREAD

**Issue by adegeo** [dotnet/wpf#4294]
Now that WPF is actively being worked on again, it would be nice to have some properties on windows to provide better control over the minimize and maximize buttons. Currently, you can't easily disable one or the other easily, especially without affecting resizing.

It's kind of silly the amount of hoops you have to jump through to get control over those two buttons. Design your complete own title bar, or jump out to p/invoke methods during window load.

- https://social.technet.microsoft.com/wiki/contents/articles/29183.wpf-disabling-or-hiding-the-minimize-maximize-or-close-button-of-a-window.aspx
- https://stackoverflow.com/questions/6523579/wpf-hide-minimize-button-of-a-window

**Comment by predavid** [comment]
Thank @adegeo  for filing the Issue, along with links to existing samples using p/invoke. 

Community - please upvote /like to help prioritize the investment. Also, if there are current apps that you are building, we'd like to hear of newer scenarios that are blocked by not having easier support for customization/resize options for min/max/close buttons.
